pkg/types/types.go

```diff
@@ -50,3 +51,11 @@ type SubmarinerSpecification struct {
 	HealthCheckMaxPacketLossCount int
 	MetricsPort                   int `default:"32780"`
 }
+
+func (subSpec *SubmarinerSpecification) GetIPFamilies() []v1.IPFamily {
```

Comment:
I think we should use the `k8snet.IPFamily` enum to be consistent with the `Endpoint` methods to avoid potentially having to translate between the two later.

Also there can be at most 2 families so we can make that explicit.

```suggestion
func (subSpec *SubmarinerSpecification) GetIPFamilies() [2]k8snet.IPFamily {
```